packages/lego-bricks/src/components/Accordion/index.tsx

```diff
@@ -84,8 +99,9 @@ export const Accordion = ({
         style={{
           height: open ? openHeight : 0,
         }}
+        onTransitionEnd={() => setTransitioning(false)}
       >
-        <div ref={childrenWrapperRef}>{children}</div>
+        <div ref={childrenWrapperRef}>{renderChildren && children}</div>
```

Comment:
Not sure if it makes much of a difference - but would it be neater to do the renderChildren check outside this div? Currently the unrendering would trigger a size change of the div, but if you hide the div in its entirety that might not be the case?

This is really minor though and just a random thought that just hit me

Reply:
I will test it and make the check outside the div if it doesn't make much difference. Adding the check there was simply the quickest and easiest when I made it. 😆